### PR TITLE
Hannah and Ylva, Movie project

### PR DESCRIPTION
**Pull request by jesinkey**
Link to my deployed project: https://hannah-och-ylva-videomys.netlify.app

My reflections on how this week's project turned out: Great! Styling was harder than we thought but it was still manageable. 

Things I'd like to have clarified or explained in more detail:Would have liked to figured out how to use the url with a path as a separate component. Also styling of svgs is still difficult and hard to understand.

### CODE REVIEW

File: code/src/components/ReturnButton.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+          className='backlink-arrow'
+          xmlns='http://www.w3.org/2000/svg'
+          viewBox='0 0 30 30'
+        >
+          <path
+            d='M27 14.5C27 7.596441 21.4035594 2 14.5 2S2 7.596441 2 14.5 7.5964406 27 14.5 27 27 21.403559 27 14.5zm-19.3388348-.353553l7.4852814-7.485282c.1952622-.195262.5118446-.195262.7071068 0l2.1213203 2.121321c.1952622.195262.1952622.511844 0 .707106L12.9644661 14.5l5.0104076 5.010408c.1952622.195262.1952622.511844 0 .707106l-2.1213203 2.121321c-.1952622.195262-.5118446.195262-.7071068 0l-7.4852814-7.485282c-.19799-.19799-.197989-.509117 0-.707106z'
+            fill='#fff'
+            fill-rule='evenodd'
+          ></path>
+        </svg>
```
Comment (leelalaim) [inline code/src/components/ReturnButton.js:18]: We recommend keeping the svg in a separate file

File: code/src/index.css [hunk 2]
```diff
@@ -1,13 +1,208 @@
...
+.thumb-container {
+  position: relative;
+  width: 100vw;
+  text-decoration: none;
+  color: white;
+}
+.thumb-image {
+  width: 100%;
+  opacity: 1;
+}
```
Comment (leelalaim) [inline code/src/index.css:36]: you can add spacing here

File: code/src/index.css [hunk 3]
```diff
@@ -1,13 +1,208 @@
...
+  height: 800px;
+}
+
+.links-wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: 10px;
+  margin-bottom: 20px;
+  margin-left: 20px;
```
Comment (leelalaim) [inline code/src/index.css:72]: You can use just margin: 10px 0 20px 20px instead

File: code/src/index.css [hunk 4]
```diff
@@ -1,13 +1,208 @@
...
+.details-title {
+  font-size: 22px;
+}
+
+/*** MEDIA QUERIES ***/
+
+@media (min-width: 768px) {
+  .thumb-container {
+    width: 25vw;
+  }
```
Comment (leelalaim) [inline code/src/index.css:140]: Nice that you have added media queries to your project 😎

File: code/src/pages/MovieDetails.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+export const MovieDetails = () => {
+  const [details, setDetails] = useState([]);
+  /*using id from the <Link> in Movie-list to make sure the fetch executes on the correct url*/
+  const { id } = useParams();
+
+  useEffect(() => {
+    fetch(
+      `https://api.themoviedb.org/3/movie/${id}?api_key=f86cf72a5f604a49cde375a4fa2c6d61&language=en-US`
+    )
```
Comment (leelalaim) [inline code/src/pages/MovieDetails.js:16]: Like another option - you can create a variable for the URL and then use it in fetch

**Review by leelalaim — APPROVED** [review]
Nice project, easy to read code and comments. Blue level reached. Well done! 🤩🦁

File: code/src/components/ReturnButton.js (hunk 1 above)
Comment (jesinkey) [reply code/src/components/ReturnButton.js:18]: Couldn't figure out how to do that, our path kept failing. 
@Karol, can you give us direction?

File: code/src/pages/MovieDetails.js (hunk 5 above)
Comment (jesinkey) [reply code/src/pages/MovieDetails.js:16]: We couldn't manage to create a variable for the url needing a path. So for consistency we made them both the same. 
@Karol, what do you think we should do?

**Review by KarolCislowski — APPROVED** [review]
Great project 👍🏻

**Pull request closed without merging**